compiler-cli/src/main.rs

```diff
@@ -526,6 +526,13 @@ fn initialise_logger() {
 }
 
 fn project_paths_at_current_directory() -> ProjectPaths {
+    let current_dir = get_current_directory().expect("Failed to get current directory");
+    let dir = get_project_root(current_dir).expect("Failed to get any gleam.toml file");
+    ProjectPaths::new(dir)
+}
+
+#[allow(dead_code)]
```

Comment:
I think we can do `#[cfg(test)]` instead

Reply:
sweet, didn't know about this attribute, thanks :)!